apps/logger/logger_118.go

```diff
@@ -0,0 +1,20 @@
+//go:build go1.18 && !go1.20
+
+package logger
```

Comment:
Can you please split the logger in a separate commit? It can even go directly to main.